corehq/apps/case_search/models.py

```diff
@@ -152,6 +154,7 @@ class CaseSearchRequestConfig:
     case_types = attr.ib(kw_only=True, default=None)
     data_registry = attr.ib(kw_only=True, default=None, converter=_flatten_singleton_list)
     custom_related_case_property = attr.ib(kw_only=True, default=None, converter=_flatten_singleton_list)
+    include_related_cases = attr.ib(kw_only=True, default=None)
```

Comment:
You should add the converted `converter=_flatten_singleton_list` otherwise this will be a list

Reply:
https://github.com/dimagi/commcare-hq/pull/32587/commits/c4d964d87763c28a1f446949ed18ce6d173b72c8